components/calendar/index.en-US.md

```diff
@@ -0,0 +1,34 @@
+---
+category: Components
+type: Presentation
+cols: 1
+english: Calendar
+---
+
+
+Container for displaying data in calendar form.
+
+## Occasion
+
+When data is in the form of date,such as schedule, timetable, prices calendar, Lunar.And this component currently support Year/Month switch.
```

Comment:
* "When data is in the form of date, such as schedule, timetable, prices calendar, Lunar calendar. This component currently support Year/Month switch."
* Not sure what prices calendar means.